src/wp-includes/class-wp-taxonomy.php

```diff
@@ -645,6 +645,7 @@ public static function get_default_labels() {
 				_x( 'A link to a tag.', 'navigation link block description' ),
 				_x( 'A link to a category.', 'navigation link block description' ),
 			),
+			'template_name'              => array( _x( 'Tag', 'taxonomy singular name' ), _x( 'Category', 'taxonomy singular name' ) ),
```

Comment:
```suggestion
			'template_name'              => array( _x( 'Tag Archives', 'taxonomy template name' ), _x( 'Category Archives', 'taxonomy template name' ) ),
```

Currently the test in the site editor shows `Tag Archives` and `Category Archives` which I think is clearer for the purpose of the template so I'd keep the existing default.

![Screen Shot 2024-05-30 at 9 36 07 am](https://github.com/WordPress/wordpress-develop/assets/519727/464243d9-84e5-4a77-abd0-69f682ab2ee8)

Reply:
Good point! This value was being [overridden in `block-template-utils.php`](https://github.com/WordPress/wordpress-develop/blob/0144dd32df52c84ba0ae62809ff36c9b08b685ca/src/wp-includes/block-template-utils.php#L175-L190), that's why for the default post category and post tag taxonomies the label with the 'Archives' suffix was still used.

But I agree that it makes sense that the default label for custom taxonomies includes the 'Archives'  suffix. I merged the change.